var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -29,27 +29,28 @@ class Rocblas(CMakePackage):
     version('3.7.0', sha256='9425db5f8e8b6f7fb172d09e2a360025b63a4e54414607709efc5acb28819642', deprecated=True)
     version('3.5.0', sha256='8560fabef7f13e8d67da997de2295399f6ec595edfd77e452978c140d5f936f0', deprecated=True)
 
-    tensile_architecture = ('all', 'gfx906', 'gfx908', 'gfx803', 'gfx900',
-                            'gfx906:xnack-', 'gfx908:xnack-', 'gfx90a:xnack+',
-                            'gfx90a:xnack-', 'gfx1010', 'gfx1011',
-                            'gfx1012', 'gfx1030')
+    amdgpu_targets = ('gfx906', 'gfx908', 'gfx803', 'gfx900',
+                      'gfx906:xnack-', 'gfx908:xnack-', 'gfx90a:xnack+',
+                      'gfx90a:xnack-', 'gfx1010', 'gfx1011',
+                      'gfx1012', 'gfx1030')
 
-    variant('tensile_architecture', default='all', values=tensile_architecture, multi=True)
+    variant('amdgpu_target', values=auto_or_any_combination_of(*amdgpu_targets))
+    variant('tensile', default=False, description='Use Tensile as a backend')
```

Comment:
`default=True` otherwise `gemm` symbols will be missing.

Reply:
I will rebase once your PR is merged.